### PR TITLE
Greek Cheat Sheet (Continue PR #1704)

### PR DESCRIPTION
**Pull request by synbit**
Continues PR #1704
IA: https://duck.co/ia/view/greek_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/language/greek.json [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+    "id" : "greek_cheat_sheet",
+    "name" : "Greek Cheat Sheet",
+    "metadata" : {
+       "sourceName" : "Omniglot",
+       "sourceUrl" : "http://www.omniglot.com/language/phrases/greek.php"
+    },
+    "aliases": [
+        "greek phrases", "english to greek", "basic greek phrases", "basic greek"
+    ],
+    "template_type": "reference",
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/language/greek.json]: This needs to be changed to `language` in order for the transliteration to appear

**Comment by mintsoft** [comment]
@moollaza Any thoughts about why this deploy is failing? I'm guessing it's because it's a replacement PR?

**Comment by mintsoft** [comment]
Interesting; the build has failed too: https://travis-ci.org/duckduckgo/zeroclickinfo-goodies/builds/111291141#L281 I'm guessing this is metadata related?

File: share/goodie/cheat_sheets/json/language/greek.json [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+          },
+          {
+             "val" : "Μπορείς να επαναλάβεις, παρακαλώ;",
+             "key" : "Can you repeat, please?",
+             "trn" : "MborIs na epanalAvis, parakalO?"
+          },
+          {
+             "val" : "Πώς λες \"X\" στα ελληνικά;",
+             "key" : "How do you say \"X\" in Greek?",
+             "trn" : "Pos les \"X\" sta HellenicA?
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/language/greek.json]: missing `"` at the end of the line here

Comment (synbit) [inline share/goodie/cheat_sheets/json/language/greek.json]: Cannot type... Thanks!

**Comment by mintsoft** [comment]
As discussed IRL we may wish to rewrite the transliteration to be more accessible

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Greek cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/greek_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Greek cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/greek_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Greek cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/greek_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Greek cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/greek_cheat_sheet_mobile.png)

**Comment by moollaza** [comment]
@synbit @mintsoft Thanks! This LGTM :+1:

**Pull request merged**